torchvision/models/detection/roi_heads.py

```diff
@@ -303,11 +330,70 @@ def paste_mask_in_image(mask, box, im_h, im_w):
     return im_mask
 
 
+def _onnx_paste_mask_in_image(mask, box, im_h, im_w):
```

Comment:
Can you write a test comparing that this function gives the same behavior as the original one?